src/EventStore.Core/Services/Transport/Http/Controllers/InfoController.cs

```diff
@@ -95,7 +97,10 @@ public class OptionStructure {
 			public string Description { get; set; }
 			public string Group { get; set; }
 			public string Value { get; set; }
-			public string[] PossibleValues { get; set; }
+			public string[] AllowedValues { get; set; }
+			public string OptionType { get; set; }
```

Comment:
Not sure if we should expose the dotnet types over JSON 🤔 could look into the types supported by https://json-schema.org/

And for the duration settings it's maybe an idea to expose those as an `ISO 8601 ABNF` duration, this way we could have a more consistent way of communicating durations which are also standardized and part of json schema.

Reply:
yeah, mapping the dotnet types is still pending. I will work on this and mark the linear issue to InProgress